python/cutlass/utils/datatypes.py

```diff
@@ -150,6 +150,10 @@ def torch_type(inp):
     import bfloat16
 
     bfloat16_available = True
+    if torch_available:
```

Comment:
Thanks for this contribution, @sophiawisdom.

I think that the best place to add this to the `_torch_to_library_dict` and `_library_to_torch_dict` would be here: https://github.com/NVIDIA/cutlass/blob/4575443d446807b8e9d7f77185452899020ac8c9/python/cutlass/utils/datatypes.py#L118-L134.

Could you please change this to add torch.bfloat16 there?

Thanks.

Reply:
Done